actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -414,9 +414,6 @@ module FormHelper
       #     form_for(record_or_name_or_array, *(args << options.merge(builder: LabellingFormBuilder)), &block)
       #   end
       #
-      # If you don't need to attach a form to a model instance, then check out
-      # FormTagHelper#form_tag.
-      #
```

Comment:
Since [passing `model: nil` to `form_with`](https://github.com/rails/rails/pull/50931) is deprecated, what do you think about repurposing this removal into guidance to explicitly pass `model: false`?

Reply:
Good idea! On top of explicitly passing `model: false`, `model` can also be omitted and it would default to `false`, so I worded my guidance accordingly.